.travis.yml

```diff
@@ -30,6 +31,8 @@ matrix:
     include:
         - php: 5.4
           env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" COVERAGE=true TEST_COMMAND="composer test-ci" SYMFONY_VERSION=2.7.*
+        - php: 7.1
+          env: SYMFONY_VERSION=4.0.*
```

Comment:
should be `env: SYMFONY_VERSION=4.0.*@dev` since there is no stable release yet

Reply:
Yes, but this will still break, because phpspec will complain: https://github.com/phpspec/phpspec/blob/master/composer.json#L29-L33

Reply:
i think we can use COMPOSER_FLAGS="--minimum-stability=dev" or something like that.